studies/wildlife/templates/scopes/wildlife.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'scopes/scope-container.html' %}
+
+{% block icon %}
+<span class="glyphicon glyphicon glyphicon-folder-open"
+  aria-hidden="true"></span>
+{% endblock %}
+
+{% block description %}
+<p>This permission grants access to your <strong>Wildlife of Our Homes</strong>
```

Comment:
Are we still using this material? I thought our rewrite of the interactions removed this stuff.